cloudify_cli/tests/commands/test_events.py

```diff
@@ -79,10 +80,8 @@ def _mock_executions_get(self, execution_id):
 
     def _mock_events_get(self, execution_id, from_event=0,
                          batch_size=100, include_logs=False):
-        if from_event >= len(self.events):
-            return [], len(self.events)
-        until_event = min(from_event + batch_size, len(self.events))
-        return self.events[from_event:until_event], len(self.events)
+        events = self._get_events_before(time.time())
+        return events[from_event:], len(events)
```

Comment:
what happend to the batch_size? previously it was used to measure the 'until_event', it seems it's missing now